android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/report/measure/MeasureReportConfiguration.kt

```diff
@@ -26,8 +26,9 @@ data class MeasureReportConfiguration(
   override var configType: String = ConfigType.MeasureReport.name,
   val id: String,
   val registerId: String,
-  val registerDate: String? = null,
+  val startPeriod: String? = "2023-01-01",
```

Comment:
This particular field represents the start date for the in-app reports. The list of months will show as back as this date. We thought renaming it to `startPeriod `would be ideal instead of `registerDate`. If left to null and if it's missed in the config the application will crash.

Reply:
Ok then we should use a constant for now and add a code comment explaining it in case we need to move it to load from a config later